### DISCUSSION TITLE
Building SCIRun against system libraries

### DISCUSSION THREAD

**Discussion by sagitter** [SCIInstitute/SCIRun#1862] [General]
Hi all.

SCIRun dowloads and builds external libraries before compiling. Any chance to have a simpler compilation against system libraries?

**Comment by dcwhite** [comment]
We had a conversation about this a while ago for another program in the CIBC suite, Seg3D. Here is [the thread](https://lists.sci.utah.edu/sympa/arc/seg3d/2015-03/msg00013.html). The relevant response still applies:

> In general, it is preferable from our point of view to build against library 
versions that we control since we have to support a broad range of platforms 
(Windows, Mac OS X and different Linux distributions). Including specific 
versions of third-party libraries allows us to maintain stable Seg3D builds 
across multiple platforms with the least amount of effort and cost.

You are free to fork the repo and write your own custom build script for this, which we could incorporate, but it's not something SCI developers will support themselves. @jessdtate, as the current technical manager, can comment further.
